src/script/lua_api/l_object.cpp

```diff
@@ -914,6 +914,58 @@ int ObjectRef::l_get_yaw(lua_State *L)
 	return 1;
 }
 
+// set_pitch(self, radians) (X-AXIS)
+int ObjectRef::l_set_pitch(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	LuaEntitySAO *co = getluaobject(ref);
+	if (co == NULL) return 0;
```

Comment:
The old code is already formatted wrong, please use either:
```C++
if (!co)
	return 0;
```
or
```C++
if (co == nullptr)
	return 0;
```
Same for your functions below.

Reply:
There are a ton of other functions in the same area that are doing the same thing I did. They will need to be fixed but I will not touch them since I do not want my pull request to spread too far.

Reply:
Yes it's fine to leave the unrelated code issues out of this PR.